scripts/build_ceo.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# this script template which creates a custom release including cluster-etcd-operator and MCO.
+# the script also will publish the image on quay.
+
+CI_TOKEN=""
+QUAY_TOKEN=""
+USER=""
+OC_PATH=/usr/local/bin/oc
+TAG="4.2"
+
+
+echo -e "Login to CI\n"
+$OC_PATH login --token="$CI_TOKEN" --server=https://api.ci.openshift.org \
+  && echo -e "Building new release\n" \
+  && sudo podman login -u="hexfusion" -p="$QUAY_TOKEN" quay.io \
```

Comment:
Do you want to use $USER here, instead of `hexfusion`?

Reply:
yep :)